env0/resource_environment.go

```diff
@@ -266,6 +266,12 @@ func resourceEnvironment() *schema.Resource {
 				Description: "set an alias for this environment in favor of running VCS commands using PR comments against it. Additional details: https://docs.env0.com/docs/plan-and-apply-from-pr-comments",
 				Optional:    true,
 			},
+			"vcs_pr_comments_enabled": {
+				Type:        schema.TypeBool,
+				Description: "set to 'true' to enable running VCS PR commands using PR comments. This can be set to 'true' (enabled) without setting alias in 'vcs_commands_alias'.",
```

Comment:
also can add docs link https://docs.env0.com/docs/plan-and-apply-from-pr-comments

```suggestion
			"vcs_pr_comments_enabled": {
				Type:        schema.TypeBool,
				Description: "set to 'true' to enable running VCS PR plan/apply commands using PR comments. This can be set to 'true' (enabled) without setting alias in 'vcs_commands_alias'.",
```

Reply:
Maybe even specifically to the Configuration section:
https://docs.env0.com/docs/plan-and-apply-from-pr-comments#configuration

Reply:
sure!